### ISSUE TITLE
Data consistency when changing visit name

### ISSUE THREAD

**Issue by chrisortman** [sparc-request/sparc-fulfillment#1013]
TLDR; 

The visit name (stored on visit group) shows up on both appointments and visits. Appointments are built just-in-time, so sometimes you can change the visit name and it will reflect on the appointment list but sometimes not. 

---
In fulfillment we see the visit name represented here

<img width="1061" alt="PastedGraphic-3" src="https://user-images.githubusercontent.com/2954/172876082-f3674767-6105-4348-b3af-f4ac520042cf.png">

That field is editable and you can change it.

The value will show up again on the participant tracker. Participant Tracker -> Click the calendar icon for one of the patients. 
On that screen you can enter data bout a visit, to do that you pick the visit from this dropdown menu.
This menu displays values from the appointments table (linked by protocols_participant_id) 

<img width="719" alt="PastedGraphic-4" src="https://user-images.githubusercontent.com/2954/172876122-d8f70a4e-18d8-4684-9531-13bc57a8d012.png">

When you *view* this page, the appointments are created. You might not do anything else, but now the appointments are there and the value of the visit name has been copied.

So now when you rename a visit you won't see the new visit names. You will also see different values depending on the participant you look at.
